src/ESA-PL-weak-copyleft-2.4.xml

```diff
@@ -137,13 +137,16 @@
                   subject to the terms and conditions of this License, to:
                </p>
                <p>
-                  use the Software; reproduce the Software by any or all
-                  means and in any or all form; Modify the Software and create
-                  works based on the Software; communicate to the public,
-                  including making available, display or perform the Software
-                  or copies thereof to the public; Distribute, sublicense,
-                  lend and rent the Software. The license grant is perpetual
-                  and irrevocable, unless terminated pursuant to Sec. 8.
+                  - use the Software;
+                  - reproduce the Software by any or all
+                  means and in any or all form;
+                  - Modify the Software and create works based on the Software;
+                  - communicate to the public, including making available,
+                  display or perform the Software or copies thereof to the
+                  public;
+                  - Distribute, sublicense, lend and rent the Software. The

```

Comment:
This will not work. It will not be displayed as bullets. This is not markdown formating. This is HTML (inside XML). So to have a list, you have to use `<list>` and `<item>` tags.

Reply:
FTR - this is on several other places of this PR, so it should be fixed there too. Personally, if you create a separate PR for Slovenia, it can be quickly merged - no problem with that. And then we can spend time tuning the list formatting.